starfish/models/squid_model.py

```diff
@@ -274,6 +296,15 @@ def get_account_balance(self, account):
         """
         return self._squid_ocean.accounts.balance(account)
 
+    def register_ddo(self, did, ddo, account):
```

Comment:
Could you please create a sequence diagram to document this flow?

Reply:
This call is not exposed to the client, it's called by the `ocean.register_update_agent_service`. 
To register a new `ocean service` or `ocean agent`. This is when I do not like the terminology of `agent`, where it's being used as a worker in the library and also as a service/server for ocean.

Reply:
Noted. OTOH, register_update_agent_service is a somewhat oblique name. Is something being registered, or updated, and it either agent or service :)